grammars/fstar.cson

```diff
@@ -249,7 +249,7 @@ repository:
         ]
       }
       {
-        begin: '`'
+        begin: '`[^%]'
```

Comment:
A better way to do this would be

~~~suggestion
        begin: '`(?!%)'
~~~

as `[^%]` won't match empty strings, or cleanly-terminated strings that start with a percentage-sign:

~~~python
empty_string  = ``
format_string = `%s`
~~~

(Full disclosure: I don't know a thing about F*)